lms/static/scripts/frontend_apps/components/test/LMSGrader-test.js

```diff
@@ -0,0 +1,73 @@
+import { act } from 'preact/test-utils';
+import { Fragment, createElement } from 'preact';
+import { mount } from 'enzyme';
+import LMSGrader, { $imports } from '../LMSGrader';
+
+describe('LMSGrader', () => {
+  const fakeStudents = [
+    {
+      userid: 'user1',
+      displayName: 'User 1',
+    },
+    {
+      userid: 'user2',
+      displayName: 'User 2',
+    },
+  ];
+  const fakeUpdateClientConfig = sinon.spy();
+  const fakeOnChange = sinon.stub();
+
+  // eslint-disable-next-line react/prop-types
+  const FakeStudentSelector = ({ children }) => {
+    return <Fragment>{children}</Fragment>;
+  };
+
+  beforeEach(() => {
+    $imports.$mock({
+      '../utils/update-client-config': fakeUpdateClientConfig,
+      './StudentSelector': FakeStudentSelector,
+    });
+  });
+
+  afterEach(() => {
+    $imports.$restore();
+  });
+
+  const renderGrader = (props = {}) => {
+    return mount(
+      <LMSGrader
+        onChangeSelectedUser={fakeOnChange}
+        students={fakeStudents}
+        {...props}
+      />
+    );
+  };
+
+  it('does not have a focused user by default', () => {
+    renderGrader();
+    sinon.assert.notCalled(fakeUpdateClientConfig);
+  });
+
+  it('clicking the next button changes to the first focused user', () => {
```

Comment:
I'm a little confused by this test as it seems to be testing behavior in `StudentSelector`...can you help me grok?

Reply:
Yep, its fixed, mostly just needs a new comment.